keyboards/pimoroni/keybow2040/info.json

```diff
@@ -0,0 +1,116 @@
+{
+    "manufacturer": "Pimoroni",
+    "keyboard_name": "Keybow 2040",
+    "url": "https://shop.pimoroni.com/products/keybow-2040",
+    "maintainer": "fanf2",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "board": "GENERIC_RP_RP2040",
+    "features": {
+	"console": true
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+    },
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x07F8",
+        "pid": "0x4784",
+    },
+    "rgb_matrix": {
+        "driver": "IS31FL3731",
+	"animations": {
+	    "alpha_mods": true,
```

Comment:
Apparently this file contains tab characters, so it looks unaligned because of some tab size mismatch.  Applying `qmk format-json` to this file should fix the formatting.

Reply:
I untabified all the json in round three